src/Sarif.Multitool/MergeOptions.cs

```diff
@@ -14,5 +14,12 @@ internal class MergeOptions : MultitoolOptionsBase
             HelpText = "Files to process (wildcards ? and * allowed).",
             Required = false)]
         public IList<string> Files { get; internal set; }
-    }
+
+	    [Option(
+		    'r',
+		    "recursive",
```

Comment:
>recursive [](start = 7, length = 9)

use 'recurse' as option and property name

Reply:
Done.

---
In reply to: [141649490](https://github.com/Microsoft/sarif-sdk/pull/627#discussion_r141649490) [](ancestors = 141649490)